src/frequenz/sdk/microgrid/component/battery_utils.py

```diff
@@ -0,0 +1,60 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Util functions for battery."""
+
+import logging
+from typing import Dict, Optional, Set
+
+from .. import get as get_microgrid
+from . import ComponentCategory
+
+_logger = logging.getLogger(__name__)
+
+
+def find_adjacent_inverter_id(battery_id: int) -> Optional[int]:
```

Comment:
Actually component graph has now methods to find successors/predecessors.
And I prefer successor/predecessor instead of `adjacent` solution as it is more error resilient.